app/src/main/java/co/develhope/meteoapp/today/TodayData.kt

```diff
@@ -0,0 +1,28 @@
+package co.develhope.meteoapp.today
+
+data class TodayData(
+    //recyclerview item elements
+
+val ora: String,
```

Comment:
i nomi dei valori, per convenzione, sono sempre in inglese e con la prima lettera in minuscolo